components/raftstore/src/store/snap.rs

```diff
@@ -1219,6 +1226,16 @@ impl SnapManager {
         Ok(v)
     }
 
+    pub fn get_temp_path_for_ingest(&self) -> String {
```

Comment:
can it be cleanup after restart?

Reply:
Yes. All files with TEMP SUFFIX will be deleted